src/main/openapi/rems.yaml

```diff
@@ -1109,37 +1117,36 @@ components:
         - license/id
         - license/enabled
         - license/archived
-        - license/description
-        - license/version
-        - license/terms
       properties:
         license/type:
           type: string
+          title: License type
           enum:
-            - link
-            - attachment
-            - text
+            - license/link
```

Comment:
important: the enum options in REMS are `link`, `attachment` and `text`, without `license/*`.

<img width="435" alt="image" src="https://github.com/user-attachments/assets/dc9447dc-dbf6-46bb-904b-7b1c9318d3d7">